django/cantusdb_project/cantusdb/settings.py

```diff
@@ -61,9 +61,11 @@
     "articles",
     "django_quill",  # to provide rich-text field for articles
     "users",
+    "debug_toolbar",
```

Comment:
Was there someway this was secure if running live? I feel like there was but I can't remember....

Reply:
In settings.py we have `DEBUG = bool(strtobool(os.getenv("CANTUSDB_DEBUG", "False")))` which I assume would assure it doesn't run live, but I'm not entirely sure where these environment variables are stored

Reply:
Fwiw my solution has been to have a separate branch - https://github.com/jacobdgm/CantusDB/tree/toolbar - and whenever I want to test something with DjDT, I run `git checkout toolbar cantusdb/settings.py` and `git checkout toolbar cantusdb/urls.py`.

I don't think we want this turned on all the time on the staging server, where `DEBUG` should always be set to `True` I think - while it's good for testing page load times, it makes things run really slowly. But I guess it's possible to turn all the features we don't want off in the browser.

All this is to say, I have this solution that you can use if you want, but there's quite possibly a better approach, so we should discuss this tomorrow.

Reply:
Ah okay, sounds good!

Reply:
I just closed #467; DEBUG is currently set to True on staging and False on Production.

I'm not sure whether to worry about having DEBUG set to True on staging. For the time being, at least, it could be quite useful to be able to check in on our query times on the ComputeCanada server (in some situations, load times are slower on CC than locally, so having the ability to test this on CC will be helpful).

A possible solution: we could start up another (limited-access) "development" server for debugging. I guess this would use up one more of our allotted IP addresses...

Reply:
Yeah, I don't think that it's an emergency in any case...just that probably the same concerns about running it in production apply to staging if those are substantially similar, which I imagine they are. 

For the longer term, something you might want to do is have the staging server no accessible to the public (eg. Cantus Ultimus staging is only accessible through the a VPN) ~~ again, I know there are bigger fish to fry right now, but maybe Katie can bring NewCantus staging into agreement with the rest of DDMAL's networking.

Reply:
@kqct?